addons/account/wizard/account_move_send.py

```diff
@@ -7,6 +7,16 @@
 from odoo.tools.misc import get_lang
 
 
+DEFAULT_PDF_TEMPLATE_NAME = 'account.report_invoice_with_payments'
+
+
+def _get_default_pdf_template_id(wizard):
+    default_template = wizard.env['ir.actions.report'].search([('report_name', '=', DEFAULT_PDF_TEMPLATE_NAME)], limit=1)
```

Comment:
Could we have two report with same name ? If not then the limit is useless right ?

Reply:
Nope, we can have 2 reports with the same report_name